### PR TITLE
Codacy fixes

### PR DESCRIPTION
**Pull request by JensWehner**
Please review carefully.

### CODE REVIEW

File: share/scripts/inverse/cma_processor.py [hunk 1]
```diff
@@ -1,115 +1,120 @@
-#!/usr/bin/env python3
+#!/ usr / bin / env python3
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Huh?

File: share/scripts/inverse/configuration_compare.py [hunk 2]
```diff
@@ -1,71 +1,70 @@
-#!/usr/bin/env python3
+#!/ usr / bin / env python3
```
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Huh?

File: share/scripts/inverse/cma_processor.py [hunk 3]
```diff
@@ -1,115 +1,120 @@
...
+new_state.state = "Running"
+new_state.parameters = es.ask()
+new_state.solutions = [0 for i in range(len(new_state.parameters))]
+new_state.comments = current_state.comments
+print("We going to State '", new_state.state, "' with parameters\n",
+      new_state.parameters, "solutions: ", new_state.solutions)
 new_state.write(args[1])
-#we need to pickle parameters as well as they are saved in a dict (string compare)
+#we need to pickle parameters as well as they are saved in a dict( \
+    string compare)
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py [hunk 4]
```diff
@@ -1,71 +1,70 @@
...
+    usage = "usage: %prog [options] conf1 conf2" parser =
+        OptionParser(usage = usage) parser.add_option(
+            "--eps", dest = "eps", metavar = "EPS",
+            help = "tolerance for mismatch", default = 1e-2)(options, args) =
+            parser.parse_args()
 
-def die(str):
-  print(str)
-  quit(254)
+                def die(string)
```
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py [hunk 5]
```diff
@@ -1,71 +1,70 @@
...
+    : print(string) quit(254)
 
-if len(args) != 2:
-  die("two configurations required as parameters")
+          if len(args) != 2 : die("two configurations required as parameters")
```
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/cma_processor.py [hunk 6]
```diff
@@ -1,115 +1,120 @@
...
-# See the License for the specific language governing permissions and
-# limitations under the License.
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
 #
 
+from optparse import OptionParser import sys from re import match import pickle
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Codacy Issue found: [invalid syntax](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py [hunk 7]
```diff
@@ -1,71 +1,70 @@
...
 
 #open both files
-try: 
-  conf1 = open(args[0])
-  conf2 = open(args[1])
+                                  try : conf1 = open(args[0])
```
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py [hunk 8]
```diff
@@ -1,71 +1,70 @@
...
-
-
 from optparse import OptionParser
 
-usage = "usage: %prog [options] conf1 conf2"
-parser = OptionParser(usage=usage)
-parser.add_option("--eps", dest="eps", metavar="EPS",
-                  help="tolerance for mismatch", default=1e-2)
-(options, args) = parser.parse_args()                  
+    usage = "usage: %prog [options] conf1 conf2" parser =
```
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indent](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py [hunk 9]
```diff
@@ -1,71 +1,70 @@
...
+        OptionParser(usage = usage) parser.add_option(
+            "--eps", dest = "eps", metavar = "EPS",
+            help = "tolerance for mismatch", default = 1e-2)(options, args) =
+            parser.parse_args()
```
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py [hunk 10]
```diff
@@ -1,71 +1,70 @@
...
-
 from optparse import OptionParser
 
-usage = "usage: %prog [options] conf1 conf2"
-parser = OptionParser(usage=usage)
-parser.add_option("--eps", dest="eps", metavar="EPS",
-                  help="tolerance for mismatch", default=1e-2)
-(options, args) = parser.parse_args()                  
+    usage = "usage: %prog [options] conf1 conf2" parser =
+        OptionParser(usage = usage) parser.add_option(
```
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/cma_processor.py [hunk 11]
```diff
@@ -1,115 +1,120 @@
...
 
-from optparse import OptionParser
-from sys import exit
-import sys
-import os
-from re import match
-import pickle
+    try : import numpy except:
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/cma_processor.py [hunk 12]
```diff
@@ -1,115 +1,120 @@
...
+    exit("Could not import numpy modules used by cma")
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Codacy Issue found: [expected an indented block](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/cma_processor.py [hunk 13]
```diff
@@ -1,115 +1,120 @@
...
 
+from optparse import OptionParser import sys import re import pickle
 
-from optparse import OptionParser
-from sys import exit
-import sys
-import os
-from re import match
-import pickle
+    try : import numpy except:
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py (hunk 9 above)
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py (hunk 8 above)
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indent](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py (hunk 5 above)
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/cma_processor.py [hunk 14]
```diff
@@ -1,115 +1,120 @@
...
+new_state.state = "Running"
+new_state.parameters = es.ask()
+new_state.solutions = [0 for i in range(len(new_state.parameters))]
+new_state.comments = current_state.comments
+print("We going to State '", new_state.state, "' with parameters\n",
+      new_state.parameters, "solutions: ", new_state.solutions)
 new_state.write(args[1])
-#we need to pickle parameters as well as they are saved in a dict (string compare)
+#we need to pickle parameters as well as they are saved in a dict( \
+    string compare)
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py (hunk 7 above)
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py (hunk 10 above)
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/configuration_compare.py (hunk 4 above)
Comment (junghans) [inline share/scripts/inverse/configuration_compare.py]: Codacy Issue found: [unexpected indentation](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/cma_processor.py [hunk 15]
```diff
@@ -1,115 +1,120 @@
...
+from optparse import OptionParser import sys import re import pickle
 
-from optparse import OptionParser
-from sys import exit
-import sys
-import os
-from re import match
-import pickle
+    try : import numpy except:
+    exit("Could not import numpy modules used by cma")
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Codacy Issue found: [expected an indented block](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

File: share/scripts/inverse/cma_processor.py [hunk 16]
```diff
@@ -1,115 +1,120 @@
...
-# See the License for the specific language governing permissions and
-# limitations under the License.
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
 #
 
+from optparse import OptionParser import sys import re import pickle
```
Comment (junghans) [inline share/scripts/inverse/cma_processor.py]: Codacy Issue found: [invalid syntax](https://app.codacy.com/app/votca-package/csg/pullRequest?prid=4536170)

**Comment by JensWehner** [comment]
Ahh clang format is triggered by the .py files

**Comment by junghans** [comment]
@JensWehner I am not a 100% sure if I like the new comment function from codacy.......

File: src/tools/csg_reupdate.cc [hunk 17]
```diff
@@ -674,17 +672,17 @@ PotentialInfo::PotentialInfo(votca::Index index, bool bonded_,
...
       string filename = potentialName + ".dist.new";
-
       try {
         dist.Load(filename);
         // for( Index i = 0; i < dist.size(); i++)
         // In the highly repulsive core (<0.1 nm) at some locations
         // an unphysical non-zero RDF value may occur,
         // so it would be better to estimate Rmin loop
         // through RDF values from Rcut to zero instead of zero to Rcut.
+        double new_min = 0.0;
```
Comment (JoshuaSBrown) [inline src/tools/csg_reupdate.cc:685]: nice :)

**Review by junghans — APPROVED** [review]

**Pull request merged**